MapboxNavigation/RouteVoiceController.swift

```diff
@@ -61,6 +54,24 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     
     /**
+     If true, a noise indicating the user is going to be rerouted will play prior to rerouting.
+     */
+    public var playRerouteSound = false
+
+    
+    /**
+     Sound to play prior to reroute. Inherits volume level from `volume`.
+     */
+    public var rerouteSound = NSDataAsset(name: "reroute-sound", bundle: Bundle.navigationUI)
```

Comment:
@1ec5 I think it's the only way we can have provide an mp3 in the framework that can be accessed in apps that import Navigation. Do you think we should use another api?